README.md

```diff
@@ -263,8 +263,10 @@ module Foo
         class Configuration < Hanami::CLI::Command
           desc "Generate configuration"
 
-          def call(*)
-            puts "generated configuration"
+          option :apps, type: :array, default: [], desc: "Generate configuration for specific apps"
```

Comment:
why add default and empty array?

Reply:
Because I want to get array type, in other words empty array or array with options